packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -64,7 +64,15 @@ const MenuLink = ({
             onClick={onClickLink}
         >
             <Icon className='header__menu-mobile-link-icon' icon={icon} />
-            <span className='header__menu-mobile-link-text'>{text}</span>
+            <span
+                className={
+                    window.location.pathname !== '/'
+                        ? 'header__menu-mobile-link-text-regular'
+                        : 'header__menu-mobile-link-text'
+                }
+            >
+                {text}
+            </span>
```

Comment:
Can you use the Text component like what you have modified in `[mobile-drawer-submenu.jsx]` and get rid of these two CSS classnames (`header__menu-mobile-link-text-regular` and `header__menu-mobile-link-text`)

Reply:
Agree, also this is going to affect style of other tabs.

Reply:
@balakrishna-binary @matin-binary 
i guess then i have to create a <Text/> component at first because modified by me <Text/> component in mobile-drawer-submenu.jsx locates in **components package**, but this one in toggle-menu-drawer.jsx in the **core package**.
Or can we import a <Text/> component from **component package** to **core package**?

Reply:
@maryiafrantsava-deriv you can use `Text` component in `core` package

Reply:
@balakrishna-binary thank you  🙏  I've done it